fonts.css

```diff
@@ -0,0 +1,22 @@
+@font-face {
+	font-family: 'MuseoSansCyrl';
+	font-display: swap;
+	src: url("../fonts/MuseoSansCyrl.eot");
+	src: local("O"), url("../fonts/MuseoSansCyrl.woff") format("woff"), 
+			url("../fonts/MuseoSansCyrl.ttf") format("truetype"), 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/baliproject/pullRequest?prid=5558708)